packages/spritesheet/src/SpritesheetLoader.ts

```diff
@@ -53,6 +53,39 @@ export class SpritesheetLoader
             return;
         }
 
+        // Check and add the multi atlas
+        // Heavily influenced and based on https://github.com/rocket-ua/pixi-tps-loader/blob/master/src/ResourceLoader.js
+        // eslint-disable-next-line camelcase
+        if (resource.data?.meta?.related_multi_packs && Array.isArray(resource.data?.meta?.related_multi_packs))
+        {
+            for (const item of resource.data.meta.related_multi_packs)
```

Comment:
Small suggestion to make this a little less verbose.
```suggestion
        const multiPacks = resource.data?.meta?.related_multi_packs;

        if (multiPacks && Array.isArray(multiPacks))
        {
            for (const item of multiPacks)
```